trace/CHANGELOG.md

```diff
@@ -12,3 +12,13 @@
   on `google-api-core` (#4221, #4280)
 
 PyPI: https://pypi.org/project/google-cloud-trace/0.16.0/
+
+## 0.17.0
```

Comment:
This should go **above** the already existing release notes (I think?)

Reply:
Done.